GooglePlayInstant/Editor/QuickDeploy/LoadingScreenGenerator.cs

```diff
@@ -246,12 +246,17 @@ private static Vector2 GetPreImportTextureDimensions(Texture2D texture)
             // play-instant-unity-plugin folder directly inside the Assets folder.
             string[] foundGuids = AssetDatabase.FindAssets(searchFilter);
 
-            if (foundGuids.Length <= 0)
+            if (foundGuids.Length == 0)
             {
                 Debug.LogErrorFormat("Failed to obtain asset from filter: {0}", searchFilter);
                 return null;
             }
 
+            if (foundGuids.Length > 1)
+            {
+                Debug.LogErrorFormat("Found multiple assets that match: {0}", searchFilter);
```

Comment:
Nit: since you still return an asset, this is more of a warning than error. But it seems very unexpected so should this be an exception?